packages/element/src/serialize.js

```diff
@@ -406,6 +410,9 @@ export function renderElement( element, context, legacyContext = {} ) {
 
 		case Consumer.$$typeof:
 			return renderElement( props.children( context || type._currentValue ), context, legacyContext );
+
+		case ForwardRef.$$typeof:
+			return renderElement( type.render( props ), context, legacyContext );
```

Comment:
Sanity check, I'm assuming you don't need to use `type.render( props, ref )` here right because `ref` would be a part of the props object in this call?

Reply:
The thinking is that we're serializing an element, if "ref" is needed/used it will probably be used in effects, events, callbacks... which are not triggered in serialization (or you're doing it wrong).

Reply:
Ya I was mostly wondering about warnings react might throw, but if testing doesn't pop up any warnings should be good then.